dev-qt/qtcore/qtcore-5.15.10.9999.ebuild

```diff
@@ -79,6 +79,11 @@ src_prepare() {
 	fi
 
 	qt5-build_src_prepare
+
+	if [[ ${PV} != *9999* ]]; then
```

Comment:
Should we just make the function a noop for 9999?

Reply:
Makes sense, I guess.